pipelines/resources/deploy-ztp-edgeclusters-no-mirror.yaml

```diff
@@ -0,0 +1,291 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: deploy-ztp-edgeclusters-no-mirror
```

Comment:
Should we call this `connected` instead of `no-mirror` ?

Reply:
it's not really connected, i.e.:  `oc get catalogsource -A
NAMESPACE               NAME                     DISPLAY                      TYPE   PUBLISHER                    AGE
openshift-marketplace   ztpfw-catalog            Disconnected Lab             grpc   disconnected-lab             7h50m
openshift-marketplace   ztpfw-catalog-certfied   Disconnected Lab Certified   grpc   disconnected-lab-certified   7h50m
`

Reply:
That's why I'm confused and what [this comment](https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable/pull/580/files#r1004004467) is about. I don't see how the GPU operator can be installed.